ci/build-webide.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
It seems like you are doing a lot of manual checking for errors in this file that would, as I understand it, be done automatically by Bash if you added
```suggestion
set -euo pipefail
```
at the top. Any reason why you avoided it?

Reply:
neglect...thx